README.md

```diff
@@ -1,7 +1,13 @@
-Usual Stuff:
+# Storyboard
 
-- bundle
-- rake db:create db:migrate db:seed
-- yarn install
-- ./bin/webpack-dev-server
-- rails s
+
+# Setup
+
+```
+gem install bundler -v 1.16.2
+bundle
+rake db:create db:migrate db:seed
+yarn install
```

Comment:
i get yelled at on Nest for not having `yarn install`d before rake tasks. Maybe doesn't effect this project, but moving this up a line might be good.

Reply:
Oh yep. I think the yarn integrity check is disabled by default now, but this could definitely block rake from running.